app/src/main/kotlin/io/homeassistant/companion/android/widgets/todo/TodoWidgetStateUpdater.kt

```diff
@@ -0,0 +1,126 @@
+package io.homeassistant.companion.android.widgets.todo
+
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.common.data.integration.friendlyName
+import io.homeassistant.companion.android.common.data.servers.ServerManager
+import io.homeassistant.companion.android.common.data.websocket.WebSocketRepository
+import io.homeassistant.companion.android.database.widget.TodoWidgetDao
+import io.homeassistant.companion.android.database.widget.TodoWidgetEntity
+import javax.inject.Inject
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.merge
+import kotlinx.coroutines.flow.onCompletion
+import kotlinx.coroutines.flow.onStart
+import timber.log.Timber
+
+/**
+ * This class allow the [io.homeassistant.companion.android.widgets.todo.TodoGlanceAppWidget] to get their
```

Comment:
Same package so I think you don't need to use the fully qualified name, but not 100% sure.
```suggestion
 * This class allow the [TodoGlanceAppWidget] to get their
```

Reply:
It probably happened when I refactor the names to fix the issue in the manifest.